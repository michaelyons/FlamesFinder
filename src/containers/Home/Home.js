import React, { Component } from 'react';

export class Home extends Component {
  getCurrentPosition = () => {
    navigator.geolocation.getCurrentPosition(function(position) {
      console.log(position);
      // do_something(position.coords.latitude, position.coords.longitude);
    });
  };

  render() {
    return (
      <div>
        <h1>poop</h1>
        <button onClick={this.getCurrentPosition}>Find Campsites Nearby</button>
      </div>
    );
  }
}

export default Home;
