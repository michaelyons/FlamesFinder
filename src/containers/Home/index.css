.findcamps-button {
  font-size: 2.4rem;
  text-decoration: none;
  background: rgb(41, 44, 48, 0.75);
  -webkit-box-shadow: 10px 10px 62px -22px rgba(0, 0, 0, 0.75);
  -moz-box-shadow: 10px 10px 62px -22px rgba(0, 0, 0, 0.75);
  box-shadow: 10px 10px 62px -22px rgba(0, 0, 0, 0);
  padding-right: 0.9rem;
  padding-left: 0.9rem;
  -webkit-border-radius: 10px;
  -moz-border-radius: 10px;
  border-radius: 10px;
}

.find-btn {
  margin-top: 13rem;
}

.fire-gif {
  width: 150px;
  margin: auto;
  display: block;
  padding-top: 17rem;
  padding-right: 11rem;
}
