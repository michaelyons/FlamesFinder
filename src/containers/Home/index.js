import React, { Component } from 'react';
import { Link } from 'react-router-dom';

import LoadingFire from '../../components/LoadingFire/LoadingFire';
import './index.css';

export class Home extends Component {
  render() {
    return (
      <div>
        <header className="App-header">
          <h1 className="App-title">FlamesFinder</h1>
        </header>
        <div className="find-btn">
          <Link to={'/campsites'} className="findcamps-button">
            Find Nearby Campsites
          </Link>
        </div>
        <LoadingFire />
      </div>
    );
  }
}

export default Home;
