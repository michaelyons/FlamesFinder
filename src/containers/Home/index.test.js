import React from 'react';
import { shallow } from 'enzyme';

import Home from './index';

describe('Footer', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Home />);
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
