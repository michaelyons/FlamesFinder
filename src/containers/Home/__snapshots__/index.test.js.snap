// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Home />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <header
          className="App-header"
>
          <h1
                    className="App-title"
          >
                    FlamesFinder
          </h1>
</header>,
        <div
          className="find-btn"
>
          <Link
                    className="findcamps-button"
                    replace={false}
                    to="/campsites"
          >
                    Find Nearby Campsites
          </Link>
</div>,
        <fireLoad />,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <h1
            className="App-title"
>
            FlamesFinder
</h1>,
          "className": "App-header",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "FlamesFinder",
            "className": "App-title",
          },
          "ref": null,
          "rendered": "FlamesFinder",
          "type": "h1",
        },
        "type": "header",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            className="findcamps-button"
            replace={false}
            to="/campsites"
>
            Find Nearby Campsites
</Link>,
          "className": "find-btn",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Find Nearby Campsites",
            "className": "findcamps-button",
            "replace": false,
            "to": "/campsites",
          },
          "ref": null,
          "rendered": "Find Nearby Campsites",
          "type": [Function],
        },
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <header
            className="App-header"
>
            <h1
                        className="App-title"
            >
                        FlamesFinder
            </h1>
</header>,
          <div
            className="find-btn"
>
            <Link
                        className="findcamps-button"
                        replace={false}
                        to="/campsites"
            >
                        Find Nearby Campsites
            </Link>
</div>,
          <fireLoad />,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <h1
              className="App-title"
>
              FlamesFinder
</h1>,
            "className": "App-header",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "FlamesFinder",
              "className": "App-title",
            },
            "ref": null,
            "rendered": "FlamesFinder",
            "type": "h1",
          },
          "type": "header",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className="findcamps-button"
              replace={false}
              to="/campsites"
>
              Find Nearby Campsites
</Link>,
            "className": "find-btn",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Find Nearby Campsites",
              "className": "findcamps-button",
              "replace": false,
              "to": "/campsites",
            },
            "ref": null,
            "rendered": "Find Nearby Campsites",
            "type": [Function],
          },
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
